base/src/main/java/butter/droid/base/torrent/TorrentBroadcastReceiver.java

```diff
@@ -28,8 +28,8 @@
 
     @Override
     public void onReceive(Context context, Intent intent) {
-        if (intent.getAction().equals(STOP)) {
-            TorrentService.stop();
+        if (STOP.equals(intent.getAction())) {
+            TorrentService.stop(context);
```

Comment:
lets make the service non-static. the static instance was useful only here

Reply:
👍